src/Maestro/Maestro.Web/Controllers/DependenciesController.cs

```diff
@@ -0,0 +1,44 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.HealthMetrics;
+using Microsoft.Extensions.Logging;
+
+namespace Maestro.Web.Controllers
+{
+    [Route("[controller]")]
+    [Route("_/[controller]")]
+    public class DependenciesController : ControllerBase
+    {
+        private readonly IRemoteFactory remoteFactory;
```

Comment:
Since this is a new file, we should follow the style guidelines at https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md